Formula/fastlane.rb

```diff
@@ -4,6 +4,7 @@ class Fastlane < Formula
   url "https://github.com/fastlane/fastlane/archive/2.183.2.tar.gz"
   sha256 "c7b3cd9f22c258dac9be1820b2f4fb46c2f476b6155744f85fb4eb61c9ebb5c7"
   license "MIT"
+  revision 1
```

Comment:
Fastlane is updated pretty often, I don' think it's needed to force users to this new version since nothing is broken.
```suggestion
```